sdk/src/invariant.ts

```diff
@@ -265,15 +265,34 @@ export class Invariant {
     )
   }
 
-  async getTick(account: IKeyringPair, key: PoolKey, index: bigint): Promise<Tick> {
-    return sendQuery(
+  async getTick(account: IKeyringPair, key: PoolKey, index: bigint): Promise<Result<Tick>> {
+    const result = (await sendQuery(
       this.contract,
       this.gasLimit,
       this.storageDepositLimit,
       account,
       InvariantQuery.GetTick,
       [key, index]
-    ) as Promise<Tick>
+    )) as any
+
+    if (result.ok) {
+      return {
+        ok: {
+          ...result.ok,
+          index: BigInt(result.ok.index),
+          liquidityChange: { v: BigInt(result.ok.liquidityChange.v) },
+          liquidityGross: { v: BigInt(result.ok.liquidityGross.v) },
+          sqrtPrice: { v: BigInt(result.ok.sqrtPrice.v) },
+          feeGrowthOutsideX: { v: BigInt(result.ok.feeGrowthOutsideX.v) },
+          feeGrowthOutsideY: { v: BigInt(result.ok.feeGrowthOutsideY.v) },
+          secondsOutside: BigInt(result.ok.secondsOutside)
+        }
+      }
+    } else {
+      return {
+        err: result.err
```

Comment:
Better will be to return an enum. These errors are our `InvarinatError` defined in contract. Conversion should be quite easy and fully generic.
1. `InvarinatError` enum is in wasm
2. The returned error is the name of the enum as string, similar to the case in the `Network` enum and the `valueOf` function